classes/AutoPanMix.sc

```diff
@@ -269,7 +273,7 @@ AutoPanMix : BaseMix {
 					// LinLin maps a range of input values linearly to a range of
 					// output values
 					panValues = Array.fill(pSlots, { arg i;
-						LinLin.kr((i % pSlots) + 1, 0, pSlots + 1, -1, 1);
+						LinLin.kr((i % pSlots) + 1, 0, pSlots + 1, -1 * panCalibration, panCalibration);
```

Comment:
Was just speaking with @forsythac about this and I want to explain how the code currently works. It basically creates two "shadow" pan slots to the far left and far right that will never be used. So if you have pSlots = 3, it will allocate "working slots" at -0.5, 0 and 0.5 and pan across those. If you have pSlots = 2, it will allocate them at -0.33 and 0.33. etc. This provides an even distribution that extends the range closer to the center for fewer slots and closer to the far sides for more slots. I'm not sure the desired behavior here, but multiplying by 0.75 will effectively just squash everything more towards the center, reducing the overall soundstage of the mix.

Also I'm concerned that the root of the problem motivating this lies within Jamulus compatibility issues and not in any way related to the panning being performed here.